app/views/admin/enterprises/_change_type_form.html.haml

```diff
@@ -27,10 +27,10 @@
               %monthly-pricing-description{ joiner: "newline" }
 
           %p.description
-            = t('.producer_description_text')
+            = t('.sell_description_text')
             %br
             %br
-            = t('.producer_description_text2')
+            = t('.sell_description_text2')
```

Comment:
I think they are @enricostano , because it was a duplicate from the first producer pack option, we were seeing twice the same text, so my first bet is they need to remain :-) But maybe Claire will have another input ;)